### PR TITLE
Added GenePattern Notebook

### PR DESCRIPTION
**Pull request by Nazeeefa**
Added [GenePattern Notebook](https://notebook.genepattern.org/) to "All tools and resources" list.

### CODE REVIEW

**Comment by bedroesb** [comment]
@Nazeeefa Thank you for your contribution! Is it possible to update your fork with our latest updates? :)

File: _data/CONTRIBUTORS.yaml [hunk 1]
```diff
@@ -182,24 +175,6 @@ Peter McQuilton:
...
-    git: mr-c
-Alan R Williams:
-    git: alaninmcr
-Chris Child:
-    git: christopherchild
-Olivier Collin:
-    git: OlivierCollin
-Vera Ortseifen:
-    git: veraort
-
```
Comment (bedroesb) [inline _data/CONTRIBUTORS.yaml:205]: This PR will overwrite this info.

**Comment by bedroesb** [comment]
@Nazeeefa Sorry for troubles, one tip I can give is to always make a feature branch that is made from an up to dat master branch.

**Comment by Nazeeefa** [comment]
@bedroesb Yes, it's updated. There's a check that seems to be unsuccessful. If anything needs to be fixed from my side, kindly let me know. Sorry for any troubles from my side, thank you for your time.

**Comment by bedroesb** [comment]
@Nazeeefa adding the tool to the [spreadsheet](https://docs.google.com/spreadsheets/d/1sITlbWYoPOchJWAUb7jHp4h-2rcQsELlPGBPZvZTkrg/edit#gid=5860261) might be the easiest solution if you want to add the tool. To solve this PR (and clean up your master branch)
I recommend you copy the chnge you made in the csv file and reset your branch using following commands:
```
(assuming you are checked out in the master branch, and the remote repo of elixir europe is called upstream):
git reset --hard upstream/master     
git push origin master --force 
```

After this you can paste you change again and push

**Comment by Nazeeefa** [comment]
@bedroesb Thanks. I already copied changes to the CSV file and tried to update repo - it says here in pull request, "This branch has no conflicts...". I will try again to fix this.

Sorry, the spreadsheet link you mentioned seems to be something else. I think you meant this: https://github.com/elixir-europe/rdmkit/blob/master/_data/main_tool_and_resource_list.csv ?

**Comment by bedroesb** [comment]
There is indeed no mergeconflict, but the conflict is in the changes you make: https://github.com/elixir-europe/rdmkit/pull/605/files 
This branch brings in an older state of the master (although you updated your master) . My proposal was to either use the commands to reset your masterbranch or to not do it through github but to just use the online spreadsheet I linked.

**Comment by bedroesb** [comment]
@Nazeeefa sorry for this mess ! I have been in your place and recognized the problem immediately!

**Comment by Nazeeefa** [comment]
No worries! Thank you for understanding and helping. It should be fine now? To be able to make a new PR, which branch should I select? sorry for a silly question, but just so I don't mess up again. Cheers!

github_pr602

**Pull request closed without merging**